mix.exs

```diff
@@ -41,7 +41,6 @@ defmodule GRPC.Mixfile do
       {:cowlib, "~> 2.12"},
       {:castore, "~> 0.1 or ~> 1.0", optional: true},
       {:protobuf, "~> 0.14"},
-      {:protobuf_generate, "~> 0.1.1", only: [:dev, :test]},
```

Comment:
@polvalente By doing this http transcoding will stop working (I remembered this now)

Reply:
I don't think it does break. It broke when I stopped using the protobuf generate, but if you look at the transcode_messages.pb.ex file, you'll see the transcoding is still there

Reply:
> I don't think it does break. It broke when I stopped using the protobuf generate, but if you look at the transcode_messages.pb.ex file, you'll see the transcoding is still there

But if the user regenerates the files from these examples would they still be present?

Reply:
Yes, as I've regenerated them. But they need to be generated with the protobut_generate plugin instead of protoc directly

Reply:
Note that the protobuf version is 0.14 in the comitted files

Reply:
> Yes, as I've regenerated them. But they need to be generated with the protobut_generate plugin instead of protoc directly

My concern is how we guide users on these issues. Since this is an example of transcoding, this caught my attention. I don't know if our documentation is clear enough for the public.

Reply:
https://github.com/elixir-grpc/grpc?tab=readme-ov-file#http-transcoding

Reply:
I think we need to bring the protobuf_generate dep back to the examples. But otherwise it seems pretty clear how to generate things

Reply:
I agree. Thank you